GDSCommon-Demo/GDSCommon-Demo/Mocks/Screens.swift

```diff
@@ -54,6 +61,18 @@ enum Screens: String, CaseIterable {
         case .gdsIntroView:
             let viewModel = MockIntroViewModel(introButtonViewModel: mockButtonViewModel)
             return IntroViewController(viewModel: viewModel)
+        case .gdsQRCodeScanner:
+            let viewModel = MockQRScanningViewModel(dialogPresenter: dialogPresenter) {
+                navigationController.popViewController(animated: true)
+            }
+            let vc = ScanningViewController(viewModel: viewModel)
+            return vc
+        case .gdsQRCodeScannerModal:
+            let viewModel = MockQRScanningViewModel(dialogPresenter: dialogPresenter) {
+                navigationController.dismiss(animated: true)
+            }
+            let vc = ScanningViewController(viewModel: viewModel)
+            return vc
```

Comment:
Would it be worth considering making an extension for the `ScanningViewController` like I did for a couple of the others to keep this switch more compact?

Reply:
We can chat about this when Amish gets back, he was to leave a comment explaining why we can't do this, and why we potentially need to look at a better way of doing this for all the screens.